detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/MultilineLambdaItParameterSpec.kt

```diff
@@ -0,0 +1,91 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.test.compileAndLint
+import org.assertj.core.api.Assertions.assertThat
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class MultilineLambdaItParameterSpec : Spek({
+    val subject by memoized { MultilineLambdaItParameter(Config.empty) }
+
+    describe("MultilineLambdaItParameter rule") {
+        context("single parameter, multiline lambda") {
+            it("reports when parameter name is implicit `it`") {
+                val findings = subject.compileAndLint(
+                    """
+                fun f() {
+                    val digits = 1234.let { 
+                        listOf(it)
+                    }
+                }""")
```

Comment:
I'd suggest you extract the source code to:

```suggestion
            val code = """
                fun f() {
                    val digits = 1234.let { 
                        listOf(it)
                    }
                }
            """
```

And use `code` to invoke `compileAndLint` afterwards. It makes the test easier to read.

Reply:
Makes sense, will fix it 👍